### PR TITLE
Add python-graphitesend-pip and python-statsd

### PR DESCRIPTION
**Pull request by v-lopez**
https://pypi.org/project/graphitesend/
https://packages.debian.org/search?searchon=sourcenames&keywords=python-statsd
https://packages.ubuntu.com/search?keywords=python-statsd

These are dependencies for https://github.com/pal-robotics/pal_statistics used to gather statistics in the graphite framework.

### CODE REVIEW

**Review by tfoote — APPROVED** [review]

**Comment by cottsay** [comment]
Looks like `python-statsd` exists on Fedora as well: https://apps.fedoraproject.org/packages/python-statsd
```yaml
  fedora: [python-statsd]
```
...should do the trick.

File: rosdep/python.yaml [hunk 1]
```diff
@@ -3811,6 +3818,9 @@ python-sqlalchemy:
     wily: [python-sqlalchemy]
     xenial: [python-sqlalchemy]
     xenial_python3: [python3-sqlalchemy]
+python-statsd:
+  debian: [python-statsd]
```
Comment (cottsay) [inline rosdep/python.yaml]:
```suggestion
  debian: [python-statsd]
  fedora: [python-statsd]
```

**Review by cottsay — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (cottsay) [reply rosdep/python.yaml]: I wish the suggestions UI had a better way to suggest additional lines, but this works...

**Comment by tfoote** [comment]
@v-lopez could you accept the suggestion from cottsay and then we can merge this?

**Comment by v-lopez** [comment]
Done

**Review by cottsay — APPROVED** [review]
Thanks!

**Pull request merged**